soominheo3268-seoul.md

```diff
@@ -0,0 +1,10 @@
+## Seoul, South Korea
+
+#### Places to eat
+  - Try some authentic Korean BBQ
```

Comment:
I ❤️ Korean BBQ! Would I find the BBQ in Seoul is similar to the Korean BBQ found in 🇺🇸 ?